### PR TITLE
add utterances comments support

### PR DESCRIPTION
**Pull request by colynn**
As we talk in here [add new comments - gitalk support ](https://github.com/daattali/beautiful-jekyll/pull/591),  add #591  @daattali 

Now add [utterances](https://github.com/utterance/utterances) comments widget support.

## Changes
add utterances-comment.html into _includes
change post.html/ page.html link to utterances-comment.html
add utterance parameters into _config.yml and README.md docs

## Demo
You can check the demo from [this website](https://colynn.github.io/aboutme/)

### CODE REVIEW

**Comment by colynn** [comment]
@daattali  If you have time, please review this pull request.

**Comment by daattali** [comment]
@colynn Thank you, I will review when I find a bit of time

**Comment by daattali** [comment]
@colynn i merged your code and made a few changes. Can you please fork the newest version of beautifuljekyll to a different repo to verify it works?

**Comment by daattali** [comment]
Nevermind, I verified it works

File: _includes/utterances-comment.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% if site.utterances.repo and site.utterances.issue-term  %}
 
 <script src="https://utteranc.es/client.js"
-        repo="{{site.utterances.repo }}"
+        repository="{{site.utterances.repo }}"
         issue-term="{{ site.utterances.issue-term }}"
         theme="{{ site.utterances.theme}}"
         label="{{ site.utterances.label}}"
```
Comment (colynn) [inline _includes/utterances-comment.html]: Adding a space before '}}'  looks more standard, although it now works

**Pull request merged**